indico/modules/users/controllers.py

```diff
@@ -696,6 +696,8 @@ def _serialize_pending_user(self, entry):
         affiliation = entry.data.get('affiliation') or ''
         email = entry.data['email'].lower()
         ext_id = f'{entry.provider.name}:{entry.identifier}'
+        avatar_url = entry.data.get('avatar_url') or get_avatar_from_name(first_name)
```

Comment:
AFAIL `entry` here is an `IdentityInfo` object from flask-multipass which never has an avatar URL, so you can always call the util here

Reply:
Removed ;)